go/common/enclave.go

```diff
@@ -26,6 +26,7 @@ type Status struct {
 	StatusCode StatusCode
 	L1Head     gethcommon.Hash
 	L2Head     *big.Int
+	EnclaveID
```

Comment:
I think this field should be included as a named field like `EnclaveID EnclaveID` - embedding it like this might be confusing or have unexpected side-effects